notebooks/official/matching_engine/similar_products_recommender_using_matching_engine.ipynb

```diff
@@ -0,0 +1,1683 @@
+{
```

Comment:
<h4>Recommend almost similar products  -&gt; Recommend the similar products (INCREASE FONT SIZE)</h4>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='104'/>

Reply:
<h4>Please find that this section is to recommend almost similar products. Recommend the similar products(this heading points to above section)</h4><p> </p>